cli/command/system/prune_test.go

```diff
@@ -11,12 +12,40 @@ import (
 func TestPrunePromptPre131DoesNotIncludeBuildCache(t *testing.T) {
 	cli := test.NewFakeCli(&fakeClient{version: "1.30"})
 	cmd := newPruneCommand(cli)
+	cmd.SetArgs([]string{})
```

Comment:
Test was failing on my machine, because `Command.Execute()` "uses the args (`os.Args[1:]` by default)", which in 
my case was `-test.v -test.run ^TestPrunePromptPre131DoesNotIncludeBuildCache$`, and thus produces;

```
prune_test.go:15: assertion failed: error is not nil: "prune" accepts no arguments.
```

Reply:
interesting :thinking: